ui/app/components/selectable-card.js

```diff
@@ -0,0 +1,41 @@
+import Component from '@ember/component';
+import { computed } from '@ember/object';
+/**
+ * @module SelectableCard
+ * SelectableCard components are card-like components that display a title, total, subtotal, and anything after they yield.
+ * They are designed to be used in containers that act as flexbox or css grid containers.
+ *
+ * @example
+ * ```js
+ * <SelectableCard @cardTitle="Tokens" @total={{totalHttpRequests}} @subText="Total" @gridContainer={{gridContainer}}/>
+ * ```
+ * @param {string} cardTitle - cardTitle displays the card title
```

Comment:
same as my previous comment the type of these params should be capitalized and include their default value (like `{String} cardTitle=null`).